packages/components/src/molecules/OrderSummary/OrderSummary.tsx

```diff
@@ -5,40 +5,36 @@ import { Label, List } from '../../'
 
 export interface OrderSummaryProps extends HTMLAttributes<HTMLUListElement> {
   /**
-   * ID to find this component in testing tools (e.g.: cypress,
+   * ID to find this component in testing tools (e.g., cypress,
    * testing-library, and jest).
    */
   testId?: string
   /**
-   * Label for the subtotal value of the order. Will only show if subtotalValue is provided.
+   * Label for the subtotal value of the order. It will only show if subtotalValue is provided.
    */
   subtotalLabel?: string
   /**
-   * Subtotal value of the order. If provided, subtotal label and value will be shown.
+   * Subtotal value of the order. If provided, a subtotal label and value will be shown.
    */
   subtotalValue?: string
   /**
-   * Label for the discount value for the order. Will only show if discountValue is provided.
+   * Label for the discount value for the order. It will only show if discountValue is provided.
    */
   discountLabel?: string
   /**
-   * Discount value for the order. If provided, discount label and value will be shown.
+   * Discount value for the order. If provided, a discount label and value will be shown.
    */
   discountValue?: string
   /**
    * Label for the total value of the order.
    */
-  totalLabel?: string
```

Comment:
@Mariana-Caetano the error is related to the `totalLabel`. Can you check it pls?
<img width="1149" alt="image" src="https://github.com/user-attachments/assets/ec51f2f4-b85c-4d94-9e86-6b2915346f09">

Reply:
My bad @hellofanny! Added the `totalLabel` again. Thanks!